operators/creating-a-python-operator.md

```diff
@@ -12,17 +12,14 @@ def my_prediction_function(input: pd.DataFrame) -> pd.DataFrame:
     return predictions
 ```
 
-{% hint style="info" %}
-All Python functions in Aqueduct by default take in Pandas DataFrames and return Pandas DataFrames. We plan to add support for other data types in the future.
-{% endhint %}
-
 When a Python function is annotated with `@op`, two things happen:
 
 1. Aqueduct packages up this code and infers any library dependencies that it might require, and prepares the code to run as a part of a workflow.&#x20;
 2. The annotated function will accept Aqueduct [Artifacts](../artifacts.md) as inputs and return Artifacts as outputs -- the return values can in turn by used by other operators.&#x20;
 
 {% hint style="info" %}
-Note that once an operator has been annotated with `@op`, it cannot be used as a "regular" Python function -- the inputs to the function must be Artifacts.
+Note that once an operator has been annotated with `@op`, it cannot be used as a "regular" Python function -- the inputs to the function can be either Artifacts or a non-Artifact.
```

Comment:
There can be multiple non-artifacts passed in, so maybe we don't use singular here? Also, saying something can "be either Artifacts or a non-Artifact" doesn't really give any new information. 

Can we sort of massage this into saying that the inputs to a function are expected to be an artifact, and if they aren't, we'll convert them into a parameter with the function argument name? It's also probably good to note that these functions always return artifacts, since that separates them from regular Python functions.